Readme.md

```diff
@@ -3,57 +3,134 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T190813)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# Spreadsheet for ASP.NET MVC - How to save and load documents from a database
 
-* [HomeController.cs](./CS/DXWebApplication23/Controllers/HomeController.cs) (VB: [HomeController.vb](./VB/DXWebApplication23/Controllers/HomeController.vb))
-* [DataHelper.cs](./CS/DXWebApplication23/Models/DataHelper.cs) (VB: [DataHelper.vb](./VB/DXWebApplication23/Models/DataHelper.vb))
-* [Index.cshtml](./CS/DXWebApplication23/Views/Home/Index.cshtml)
-* **[SpreadsheetPartial.cshtml](./CS/DXWebApplication23/Views/Home/SpreadsheetPartial.cshtml)**
-<!-- default file list end -->
-# Spreadsheet - How to save and load documents from a database
-
-## Steps to implement:
-1) Create a helper class that returns the Spreadsheet settings. Make sure that you specify the **Name** and the **CallbackRouteValues** properties:
-```cs
- public static class SpreadsheetSettingsHelper {
+This example demonstrates how to configure the [Spreadsheet extension](https://docs.devexpress.com/AspNetMvc/17113/components/spreadsheet) to save and load documents from a database.
```

Comment:
A potential issue here: you say "to save and **load** documents **from** a database"

Thing is, that you can **load** documents **from** a database. At the same time, you can **save** documents **to** a database.

That is, two different prepositions designate two different information flow directions. How can we resolve this issue?

1. The simplest way is to join "save/load" and "to/from" through a slash character.

```suggestion
This example demonstrates how to configure the [Spreadsheet extension](https://docs.devexpress.com/AspNetMvc/17113/components/spreadsheet) to save/load documents to/from a database.
```

OR

2. Alternatively, we can find a synonym that can logically generalize the both operations. That would be "store" or "persist" in this scenario. If you find "to store documents in a database" too abstract, you may stick to the solution with slash characters. However, I don't see any fundamental issues in the second scenario.

```suggestion
This example demonstrates how to configure the [Spreadsheet extension](https://docs.devexpress.com/AspNetMvc/17113/components/spreadsheet) to store documents in a database.
```

I'm OK with either variant. However, you may come up with a better variant. Who knows.)